i18n/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -1138,4 +1138,9 @@
     <string name="download_speed_limit_hint">Set to 0 to disable the speed limit.</string>
     <string name="pref_mpv_scripts">Enable MPV scripts</string>
     <string name="pref_mpv_scripts_summary">Needs external storage permission.</string>
+    <string name="torrentserver_is_running">Torrent Server is running</string>
+    <string name="pref_category_torrentserver">TorrentServer Preferences</string>
```

Comment:
You forgot a space. Or was this intentional?

Reply:
yep i forget a space